WindowsFormsApp3/Form1.cs

```diff
@@ -109,10 +111,12 @@ void CreateFromWorld(IWorldspaceGetter wrld)
                     bool black = false;
                     foreach (var cell in subblock.Items)
                     {
+                        if (cell.Grid == null) continue;
```

Comment:
The nullability errors being turned on meant that certain member accesses here were now giving compiler errors (because they were potentially null and you weren't checking for that)

Just upgraded them to check for null first and continue if a cell ever didnt have a grid?